src/NetscriptFunctions/Gang.ts

```diff
@@ -297,6 +297,22 @@ export function NetscriptGang(): InternalAPI<IGang> {
         ...member.getAscensionResults(),
       };
     },
+    getInstallResult: (ctx) => (_memberName) => {
```

Comment:
Why does the netscript function `getInstallResult` do something different from the gang function `getInstallResults`? It feels like either one of those is misnamed, or misimplemented.

Reply:
If you have issues with either of these then I would need to change getAscensionResult and other methods too, because I just matched the behavior and naming style of those.

Reply:
It's not though? `getAscensionResult` is returning literally the result of `member.getAscensionResults`, except with `earnedRespect` tacked on. Here, you're doing this transformation where the gang function returns the post-install result but the netscript function returns a ratio.

Reply:
Ah. that's just shifted over, look at the function body of getAscensionResults on GangMember. I shifted that over because the install usage needs the raw points while the player exposed method needs the multipliers. getAscensionResult doesn't need to care about raw points.

Reply:
Yes, that's my entire point. Gang has `getAscensionMultsAfterAscend` for the raw values (not entirely needed, because it's only used in one place, but it's fine) and `getAscensionResults` returns the ratios. And then this matches the NS function.

My argument is that the gang function you're adding ought to match the NS function.

Reply:
Honestly I don't like how it looks either but I couldn't think of a better way and it's the closest possible match to existing behavior in the twin function (getAscensionResult).

Reply:
How about `getPostInstallPoints`?

Reply:
Done.